dev/tools/proto-to-mapper/main.go

```diff
@@ -268,6 +268,7 @@ func (v *visitor) writeTypes(out io.Writer, msg protoreflect.MessageDescriptor)
 					goType = "map[string]int64"
 				} else {
 					fmt.Fprintf(out, "// TODO: map type %v %v\n", keyKind, valueKind)
```

Comment:
Do we want to include field or goFieldName or jsonName here?

Reply:
Good call - will do!